### PR TITLE
Tags update

### PR DESCRIPTION
**Pull request by dhrubpaul**

### CODE REVIEW

File: tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md [hunk 1]
```diff
@@ -38,11 +37,17 @@ pip --version
 
 [ACCORDION-BEGIN [Step 2: ](Install SAP AI API Client SDK)]
 
-Python packages have format `.whl` or `.tar.gz`, both can be installed in the same way. `SAP-AI-API-Client-SDK`  is python package which will enable us to talk to SAP AI API.
-Follow the steps to install the same
+`SAP-AI-API-Client-SDK`  is python package which will enable us to talk to SAP AI API.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md:40]: Please don't use the "royal we" unless you are referring to SAP. Say "you" instead when you are addressing the user.

`SAP-AI-API-Client-SDK`  is python package - There is a missing article and there should be a comma before "which".

File: tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md [hunk 2]
```diff
@@ -97,66 +103,78 @@ jupyter notebook
...
+Get service key for your SAP AI Core. [Read How to create service key for SAP AI Core.](https://help.sap.com/viewer/2d6c5984063c40a59eda62f4a9135bee/LATEST/en-US/7323ff4e37ba41c198b06e9669b80920.html)
 
-Follow the [provisioning steps](https://help.sap.com/viewer/product/AI_CORE/CLOUD/en-US) to get an SAP AI API service instance key.
+> If you have not provisioned SAP AI Core. Read [provisioning steps](https://help.sap.com/viewer/2d6c5984063c40a59eda62f4a9135bee/LATEST/en-US/38c4599432d74c1d94e70f7c955a717d.html) to provision SAP AI Core service.
 
-> If you already have an SAP AI API service instance you can get the service key from your SAP BTP cockpit:
-**SAP BTP cockpit > subaccount > Instances and Subscriptions > Instances > Credentials**
-
-The content of your SAP AI API service file should look similar to this:
+Here an example service key file. The content of your SAP AI API service file should look similar to this:
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md:157]: Please add a verb.

File: tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md [hunk 3]
```diff
@@ -187,27 +204,29 @@ ai_api_client = AIAPIV2Client(
 
     !![new cell](img/jupyter/new-cell.png)
 
-2. Paste the code snippet and click the arrow *(Alternative: **Run** > **Run Cells**)*.
+2. Paste code snippet and click the arrow to execute. *(Alternative: **Run** > **Run Cells**)*.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md:207]: Why did you remove the article of "code snippet"?

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by dhrubpaul** [comment]
@MichaelCzcz 
As requested the changes have been made.
Kindly review.

-Dhrub
